packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -107,20 +111,28 @@ export default class {
   }
 
   async getAllRemotesInfo() {
-    const remotes = await this._remotes.get()
+    const remotesInfo = this._remotesInfo
+    await asyncMap(this._remotes.get(), async remote => {
+      const getInfo =
```

Comment:
Does it really help to create intermediary functions?

Reply:
It's needed in order to not duplicate the logic.

Reply:
Why? Is `getInfo` called twice?